exercice2.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+$nbDeChien = readline("Combien avez-vous de chien?");
+// $chiens = [
+//     0=>[
+//         'nom'=>'Dogo1',
+//         'race'=>'Teckel'
+//     ],
+//     1=>[
+//         'nom'=>'Dogo2',
+//         'race'=>'Husky'
+//     ]
+// ];
+
+$chiens = [];
+
+
+
+for($i=0; $i < $nbDeChien; $i++){
+    $nom = readline("Quel est son nom?");
+    $chiens[$i]['nom'] = $nom;
+    $race = readline("Quelle est sa race?");
+    $chiens[$i]['race'] = $race;
+}
```

Comment:
Maybe you can improve things alittle bit here by asking "Quel est le nom du chine N°n ?", otherwise the experience can be a little confusing, as the user is asked the same question multiple times.

Could you find a way to do this ?

Reply:
You used a `for` loop here. Could you have achieved the same result with another loop type ? If yes, which one ?

Reply:
Il est possible de le faire avec la boucle while 

```
 $i = 1;
while($i <= $nbDeChien){

    $nom = readline("Quel est le nom du chien $i ?");
    $chiens[$i]['nom'] = $nom;
    $race = readline("Quelle est la race du chien $i ?");
    $chiens[$i]['race'] = $race;
    $i++;
}
```

Reply:
Effectivement :+1: